torch/_library/fake_class_registry.py

```diff
@@ -99,10 +99,26 @@ def _check_valid_flat_script_obj(flat_x):
             )
 
 
-def to_fake_obj(fake_mode, x: torch.ScriptObject) -> FakeScriptObject:
+def tracing_with_real(x: torch.ScriptObject) -> bool:
+    if not hasattr(x, "tracing_mode"):
```

Comment:
Do you think people want all instances of an e.g. TensorQueue to trace with real, or they might want to do different behavior for specific TensorQueue?

If it's all then tracing_mode should be a classmethod. This might be the simplest for now.